tests/test_leggett_platt_mlrm.py

```
@@ -0,0 +1,975 @@
+    entry.add_to_hass(hass)
+    return entry
+
+
+class TestLeggettPlattMlrmController:
+    """Test Leggett & Platt MlRM controller."""
+
+    async def test_control_characteristic_uuid(
+        self,
+        hass: HomeAssistant,
+        mock_leggett_mlrm_config_entry,
+        mock_coordinator_connected,
```

Ruff (0.14.13)

Unused method argument: `mock_coordinator_connected`

(ARG002)